force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,20 +352,38 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
-        
+        private String name;
+        private Integer age;
         // Constructor to initialize the name and age
-    
-        // Method introduceYourself that returns the details of the person        
-        // Return a string in the format "Hello, my name is <name> and I am <age> years old."
+        public Person (String name, Integer age){
+            this.name = name;
+            this.age = age;
+        }
+        // Method introduceYourself that returns the details of the person
+        public String introduceYourself(){
+            String intro = 'Hello, my name is ' + name + ' and I am ' + age + ' years old.';
+            // Return a string in the format "Hello, my name is <name> and I am <age> years old."
+            return intro;
+        }
         // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
-
         // Getter method that returns the name of the person
-
+        public String getName(){
+            return name;
+        }
         // Getter method that returns the age of the person
-
+        public Integer getAge(){
+            return age;
+        }
         // Setter method that sets the name of the person
-
+        public String setName(String name){
+            this.name = name;
+            return name;
+        }
         // Setter method that sets the age of the person
+        public Integer setAge(Integer age){
+            this.age = age;
+            return age;
+        }
```

Comment:
Typically setters don't return anything.  If you wanted them to I would have  them return the whole object.  Something like this. 

```
 public Person setName(String name) {
            this.name = name;
            return this;
        }
        ```